src/components/pricing/select-plan.tsx

```diff
@@ -74,41 +64,7 @@ const SelectPlan: React.FunctionComponent<SelectPlanProps> = ({
                 </ul>
               </div>
             </div>
-            <div className="py-8 px-6 text-center bg-gray-50 lg:flex-shrink-0 lg:flex lg:flex-col lg:justify-center lg:p-12">
-              <p className="text-lg leading-6 font-medium text-gray-900">
-                One low price...
-              </p>
-              <div className="mt-4 flex items-center justify-center text-5xl leading-none font-extrabold text-gray-900">
-                {prices ? (
-                  <span>${prices.annualPrice.price}</span>
-                ) : (
-                  <span>$ ---</span>
-                )}
-                <span className="ml-3 text-base leading-7 font-medium text-gray-500">
-                  USD
-                </span>
-              </div>
-
-              <div className="mt-6">
-                <div className="rounded-md shadow">
-                  <button
-                    onClick={onClickCheckout}
-                    disabled={!prices || viewer?.is_pro}
-                    className={`${
-                      !prices || loading || viewer?.is_pro
-                        ? 'opacity-40'
-                        : 'opacity-100'
-                    } w-full flex items-center justify-center px-5 py-3 border border-transparent text-base leading-6 font-medium rounded-md text-white bg-gray-900 hover:bg-gray-800 focus:outline-none focus:shadow-outline transition duration-150 ease-in-out`}
-                  >
-                    {prices && !loading
-                      ? viewer?.is_pro
-                        ? `Already a Member!`
-                        : `Get Access`
-                      : '--'}
-                  </button>
-                </div>
-              </div>
-            </div>
+            {children}
```

Comment:
felt like this section would probably be fine composed in instead of passing down props